en/development/configuration.rst

```diff
@@ -122,6 +122,23 @@ Asset.cacheTime
     <http://php.net/manual/en/function.strtotime.php>`_ can take.
     The default is ``+1 day``.
 
+Using a CDN
+------------------------------------
```

Comment:
too long header line?

Reply:
Too long is ok, too short causes build failures.